guides/source/active_record_multiple_databases.md

```diff
@@ -196,6 +196,21 @@ Note that there is no command for creating the database users, and you'll need t
 to support the readonly users for your replicas. If you want to create just the animals
 database you can run `bin/rails db:create:animals`.
 
+## Connecting to Databases without Managing Schema and Migrations
+
+If you would like to connect to an external database without any database mangement tasks such as schema management, migrations, seeds, etc. you can use the per database config option `database_tasks: false` by default it is considered true.
```

Comment:
Can you wrap this documentation to 80 characters?

Reply:
Done. Seemed like the rest of the file used ~100 though.